### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by 1mehdifaraji** [Leocardoso94/animated-number-react#38]
I want the numbers to start animating when I scroll to the viewport of numbers . So when I scroll down and see the numbers it should start animating from zero but now when the page is reloaded and I scroll to the numbers viewport , the animation has been ended already .

**Comment by EvanDaley** [comment]
I'm also curious about this feature request. Is there a good way to do this manually?

**Comment by 1mehdifaraji** [comment]
> I'm also curious about this feature request. Is there a good way to do this manually?

Yes but you need to use another package just for this tiny feature and use lazy load with this package .
